wasm/src/account/private_key.rs

```diff
@@ -100,6 +100,15 @@ impl PrivateKey {
         Signature::sign(self, message)
     }
 
+    /// Convert a message to field representation and sign the fields with a private key
+    ///
+    /// @param {PrivateKey} private_key The private key to sign the message with
+    /// @param {String} message The message to sign
+    /// @returns {Signature} Signature of the message
+    pub fn sign_fields(&self, message: &str) -> Result<Signature, String> {
```

Comment:
What this method actually does is sign an Aleo type and will fail if it's NOT a valid Aleo type. 

The conversion into field happens under the hood and isn't really exposed to users, and generally functions presented in an API shouldn't describe the imperative process they're encapsulating, they should describe the top level action being done.

Here what we're accomplishing is signing a valid Aleo Value type (i.e. a Plaintext, Record, or Future) and so it should be named that. Sign_fields would indicate you're signing an array of fields. 

```suggestion
    pub fn sign_value(&self, message: &str) -> Result<Signature, String> {
```